### PR TITLE
`perllol`: Remove indirect object syntax in `Dumpvalue` example

### PR DESCRIPTION
**Pull request by rwp0**
Fix `dumpValue` method call syntax as well.

Change the variable name to `$prettify`
because the adverb doesn't make sense
without the indirect syntax.

The documentation shows correct examples:

https://perldoc.perl.org/Dumpvalue

### CODE REVIEW

**Comment by mauke** [comment]
I'd suggest rewording the commit message as "perllol: remove indirect object syntax in Dumpvalue example" or similar.

**Comment by rwp0** [comment]
> I'd suggest rewording the commit message as "perllol: remove indirect object syntax in Dumpvalue example" or similar.

Done, though I'd prefer shorter commit messages.

**Review by tonycoz — APPROVED** [review]

**Comment by rwp0** [comment]
@mauke can we merge it now (after the change and an approval) ?
